cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/analyzer/ExcludedBrokersForLeadershipTest.java

```diff
@@ -112,6 +116,38 @@
     p.add(params(5, RackAwareDistributionGoal.class, excludeB1, OptimizationFailureException.class, DeterministicCluster.rackAwareUnsatisfiable(),
                  deadBroker0, null));
 
+    // ============MinTopicLeadersPerBrokerGoal============
+    Properties configOverrides = new Properties();
+    configOverrides.put(AnalyzerConfig.TOPICS_WITH_MIN_LEADERS_PER_BROKER_CONFIG, TestConstants.TOPIC_MUST_HAVE_LEADER_REPLICAS_ON_BROKERS);
```

Comment:
If this is a common property that we would like to use in tests, let's move this to `AnalyzerUnitTestUtils#goal`, so that we can avoid changes to handle `goal` creation (1) with and (2) without this property separately in tests.

Reply:
I don't think this constant `TOPIC_MUST_HAVE_LEADER_REPLICAS_ON_BROKERS` is common property. Some tests use it to test this new hard goal. However, in some other test cases, this constant variable is not used since in these cases, we want to test how this new goal handles a cluster model with no topic names such as `TOPIC_MUST_HAVE_LEADER_REPLICAS_ON_BROKERS`

Reply:
sounds good -- thanks for the clarification